mw-webapp/src/logic/wayPage/wayStatistics/WayActiveStatistic.tsx

```diff
@@ -94,9 +94,8 @@ export const WayActiveStatistic = observer((props: WayStatisticProps) => {
   const dayReportsReversed = [...props.dayReports].reverse();
   const startDate = dayReportsReversed[0] ? dayReportsReversed[0].createdAt : props.wayCreatedAt;
   const lastDate = props.dayReports[0] ? props.dayReports[0].createdAt : props.wayCreatedAt;
-  const startDateLastWeek = startDate <= DateUtils.getLastDate(AMOUNT_DAYS_IN_WEEK, lastDate)
-    ? DateUtils.getLastDate(AMOUNT_DAYS_IN_WEEK, lastDate)
-    : startDate;
+  const lastDateWeek = DateUtils.getLastDate(AMOUNT_DAYS_IN_WEEK, lastDate);
```

Comment:
Could we use value of lastWeekDate - const from 49 line? Or it's 2 different values?

Reply:
yes it's 2 different values

Reply:
var lastWeekDate returns last week date from current date 
var lastDateWeek Reportslast week date from last date reports

Reply:
renamed variable to lastDateMonthReport